.github/workflows/bahmni_lite_docker_env.yaml

```diff
@@ -0,0 +1,69 @@
+name: Deploy to Bahmni Lite Docker Env
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'bahmni-proxy/**'
+      - '.github/workflows/bahmni_lite_docker_env.yaml'
+      - '.github/workflows/proxy_image_build_publish.yml'
```

Comment:
`bahmni-lite` folder should also be tracked, so that any changes to docker-compose / .env would be reflected